src/cli/mod.rs

```diff
@@ -90,35 +91,51 @@ enum Commands {
 }
 
 pub async fn init() -> ExitCode {
-	// Start a new CPU profiler
-	#[cfg(feature = "performance-profiler")]
-	let guard = pprof::ProfilerGuardBuilder::default()
-		.frequency(1000)
-		.blocklist(&["libc", "libgcc", "pthread", "vdso"])
-		.build()
-		.unwrap();
-	// Parse the CLI arguments
 	let args = Cli::parse();
```

Comment:
Since the CLI args get parsed later, this line should be below the chunk that got moved